tests/test_exporters.py

```diff
@@ -135,14 +144,7 @@ def test_export_binary(self):
         self.assertEqual(expected, exporter.export_item(value))
 
     def test_other_python_types_item(self):
```

Comment:
a nitpicking: how about renaming the methods to `test_nonstring_types_item` to match the item description?